wiki/Beatmaps/TAG/en.md

```diff
@@ -0,0 +1,16 @@
+---
+stub: true
+tags:
+  - multiplayer
+  - co-op
+---
+
+# TAG
+
+**TAG** is a term used to describe a type of [beatmap](/wiki/Beatmaps) that is designed for more than one person to play at the same time in [multiplayer](/wiki/multi) mode, using the [Tag Coop or Tag Team Vs](/wiki/Multi#tag-coop-/-tag-team-vs) mode.
```

Comment:
```suggestion
**TAG** is a term used to describe a [multiplayer](/wiki/multi) game mode named [Tag Coop or Tag Team Vs](/wiki/Multi#tag-coop-/-tag-team-vs), where players try to complete a [beatmap](/wiki/Beatmaps) cooperatively. Beatmaps that are specifically designed for this mode are usually referred to as **TAG beatmaps/difficulties**.
```
I felt the need to rewrite this line since TAG diffs are a thing because of that very multiplayer mode